aries-site/src/pages/design-tokens/overview.mdx

```diff
@@ -0,0 +1,85 @@
+import { Box, Text } from 'grommet';
+import { Example } from '../../layouts';
+import { BetaNotification } from '../../examples';
+
+<BetaNotification />
+
+## What is a design token?
+
+Design tokens codify the design decisions that make up a design system's visual style. Design tokens replace hard-coded values, such as hex codes for color or pixel values for spacing, with clear names.
+
+Tokens can represent values such as colors, typography, spacing, motion and more.
+
+An HPE Design Token consists of two main parts:
+
+1. A self-documenting name that represents the coded value: “hpe.color.brand".
+1. An associated value: #01A982.
+
+Additionally, a design token can have these optional properties:
+
+1. Type: A category for the token's value (e.g., color, dimension, shadow).
+1. Description: Concise text to describe how the token should be used.
+
+To know more about token naming, visit [How to read tokens](tokens/how-to-read-tokens).
+
+TO DO: Add visual
+
+## Why are design tokens important?
+
+Design tokens establish a source of truth for HPE's visual style in a tech-agnostic format, enabling a truly consistent, unified experience across applications.
+
+Token names are purpose-driven, rather than hard-coded values, which provides the flexibility for token values to evolve over time with our visual style. When tokens are implemented across design and development, style updates will propagate consistently across applications simply by pulling in the latest version of the tokens.
+
+Design tokens allow us to:
+
+- Codify design decisions
+- Enable teams across various tech stacks to build efficiently and consistenly with the HPE Design System
+- Replace manual maintenance methods with faster automated processes that keep design and development synchronized
+- Build a shared language across design and development
+
+## Types of tokens
+
+Our tokens can be grouped into 3 levels: primitive, semantic, and component. The naming and intended use of a token get more specific as you move from the primitive to semantic to component level. This tiered system of tokens exists to enhance token consumption, management, and maintenance.
+
```

Comment:
Would a statement, maybe in a notification callout, saying "When consuming design tokens, begin with the most specific token level available. Look first to see if a component token exists, if not available then use semantic tokens as a fallback."

Reply:
Done.